src/crimson/os/seastore/extent_placement_manager.h

```diff
@@ -351,9 +355,8 @@ class ExtentPlacementManager {
     LOG_PREFIX(ExtentPlacementManager::delayed_alloc_or_ool_write);
```

Comment:
please modify this function's description, lba update is removed. and this function just do delayed alloc, no ool_write, right?  can we just rename it delayed_allocate?

Reply:
I'd actually like to leave that as is.  Once we start trying to optimize for larger RGW writes, I think we'll actually want to do ool writes here for large write sizes.